activerecord/lib/rails/generators/active_record/model/model_generator.rb

```diff
@@ -51,18 +51,26 @@ def parent_class_name
           options[:parent] || determine_default_parent_class
         end
 
-        def determine_default_parent_class
-          application_record = nil
-
+        def application_record_exist?
           in_root do
-            application_record = if mountable_engine?
-              File.exist?("app/models/#{namespaced_path}/application_record.rb")
+            if mountable_engine?
+              return File.exist?(application_record_file_name)
```

Comment:
Yes. Because `in_root` is not returned as a return value the process was executed in the block, it was used early return. 
However, when the early return, so seem to stack in the `thor` is not properly released, and then modified to use the variable.